skytap/convert.go

```diff
@@ -0,0 +1,29 @@
+package skytap
+
+// String returns a string value for the passed string pointer.
+// It returns the empty string if the pointer is nil.
+func String(s *string) string {
```

Comment:
This set of functions seems gratuitous and unnecessary

Reply:
@ja6a I understand what they are. please make them private and clean up any naming conflicts.